server/submission/queries.ts

```diff
@@ -0,0 +1,34 @@
+import { Submission } from 'server/models';
+
+export const createSubmission = async (inputValues, actorId = null) =>
+	Submission.create(
+		{
+			pubId: inputValues.pubId,
+			status: inputValues.status,
+		},
+		{ actorId },
+	);
+
+export const updateSubmission = async (inputValues, updatePermissions, actorId = null) => {
```

Comment:
This function (and in general, all new functions) should have type annotations on its arguments:

```ts
(inputValues: Partial<Submission>, updatePermissions: keyof Submission, actorId: null | string = null) => ...
```

I would consider renaming `inputValues` to `patch` or `update` and `updatePermissions` to `allowedFields`. I know these names are in heavy rotation in older code, but they don't really make sense — "permissions" in particular points to a totally different concept in our codebase.

Also, since there's only ever going to be one updatable field for the forseeable future, I think it might be simpler to have an `updateSubmissionStatus` function instead of this one.

Reply:
Amen. I am inclined to do this as a homework assignment across the board, but it's not pressing. A little back end semantic meaning tune-up could go a long way for the Qwelian's and Ben's of the world, just starting to get familiar with the back end.